apps/modernization-ui/src/apps/patient/add/extended/inputs/identification/IdentificationMultiEntry.tsx

```diff
@@ -0,0 +1,40 @@
+import { IdentificationEntry } from 'apps/patient/data/entry';
+import { IdentificationView } from './IdentificationView';
+import { MultiValueEntry } from 'design-system/entry/multi-value/MultiValueEntry';
+import { internalizeDate } from 'date';
+import { Column } from 'design-system/table';
+import { IdentificationEntryFields } from 'apps/patient/data/identification/IdentificationEntryFields';
+
+const defaultValue: Partial<IdentificationEntry> = {
```

Comment:
suggestion (non-blocking):  Would it make sense to put the defaultValue with the definition of the type?